packages/functions/src/processor/piece-buffering.js

```diff
@@ -25,9 +24,7 @@ Sentry.AWSLambda.init({
 async function pieceBufferringWorkflow (sqsEvent) {
   const { storeClient, queueClient } = getProps()
   const pieceRecords = sqsEvent.Records.map(r => r.body)
-
-  // TODO: confirm group ID uniqueness
-  const groupId = sqsEvent.Records[0].attributes.MessageGroupId
+  const groupId = sqsEvent.Records[0].attributes.MessageGroupId || 'did:web:web3.storage'
```

Comment:
Could different messages have different `MessageGroupId` and if so isn't this logic flawed ?

Reply:
Please use config variable instead of hard coding did so we can easily pass / change it if needed.

Reply:
yes, but this is now dependent on https://github.com/web3-storage/w3filecoin/issues/49

We need to send one or deployed queue fails, and if we end up with the buffering we are not going to look into those. So punting this on how we will do this part